src/chains/arbitrum/vm/opcodes/block/coinbase.ts

```diff
@@ -0,0 +1,24 @@
+import { OpcodeWithoutPlaygroundLink } from '@/chains/types';
+
+export const coinbase: OpcodeWithoutPlaygroundLink = {
+  number: 0x41,
+  name: 'coinbase',
+  description:
+    'The concept of `block.coinbase` is not applicable in the context of Optimistic rollups because only the sequencer is responsible for creating and finalizing blocks. The sequencer does not receive any direct mining rewards for its role in block creation so there is no need for a coinbase address.',
+  minGas: 0, // TODO: Couldn't find this value
```

Comment:
I couldn't find anything on this...

Reply:
According to https://www.evm.codes/#41?fork=shanghai that opcode costs 2 gas. This is confirmed by the execution-specs repo [here](https://github.com/ethereum/execution-specs/blob/master/src/ethereum/shanghai/vm/instructions/block.py#L84) where it charges `GAS_BASE`, which is defined as 2 gas [here](https://github.com/ethereum/execution-specs/blob/master/src/ethereum/shanghai/vm/gas.py#L24)

Reply:
Oops sorry I didn't realize this was in the arbitrum directory 🤦

Yea let's leave a TODO comment, and for anything we can't find that we leave as TODO let's create an issue to track